api/activetigger/functions.py

```diff
@@ -738,8 +760,8 @@ def request_ollama(endpoint: str, request: str, model: str = "llama3.1:70b"):
     if response.status_code == 200:
         try:
             return {"success": response.json()["response"]}
-        except:
-            return {"error": "Error in the content"}
+        except Exception as e:
+            return {"error": f"Error in the content: {e}"}
```

Comment:
+1, 

Be mindful the traceback may contain sensitive information if verbose traceback are activate (e.g if a local variable contain an API Key, verbose tb could contain the value of the local variable.

Reply:
Ok, i see the point